### DISCUSSION TITLE
Possible to grep in other window

### DISCUSSION THREAD

**Discussion by gamb** [minad/consult#999] [Ideas]
I'd like something similar to the `other-window` prefix where grepping (`consult-grep`) keeps the current buffer intact and instead I can navigate to / preview results in another window.

Naively I tried:

```elisp
(let ((consult--buffer-display #'switch-to-buffer-other-window))
      (consult-ripgrep dir initial))
```

But I don't think this variable is compatible with `consult-grep` and co.

**Comment by minad** [comment]
Yes, this is not possible right now. Buffer preview is somewhat limited, but a possible improvement is planned. Discussion here: https://github.com/minad/consult/discussions/855
